.gitignore

```diff
@@ -19,3 +19,5 @@ MANIFEST
 dask-worker-space
 .coverage.*
 .tmp-test-dir-with-unique-name/
+.spyproject/
+chet_dev/
```

Comment:
@ChetGoerzen would you mind removing these? It's best to avoid commiting things to these files in PRs. These are usually things you'd want ignored in all your repos, so you can set them in a global `.gitignore`.